lib/errors/index.js

```diff
@@ -51,7 +51,7 @@ function IgnitionError(options) {
         }
 
         Object.getOwnPropertyNames(options.err).forEach(function (property) {
-            if (['errorType', 'name', 'statusCode', 'message', 'level'].indexOf(property) !== -1) {
+            if (['errorType', 'name', 'statusCode', 'message', 'level', 'code'].indexOf(property) !== -1) {
```

Comment:
This is wrong I think - need to only skip `code` if it exists in the options

Reply:
yeah needs to be conditional